go/kbfs/libkbfs/folder_branch_ops.go

```diff
@@ -7641,7 +7714,7 @@ func (fbo *folderBranchOps) unstageAfterFailedResolution(ctx context.Context,
 	ctx = newLinkedContext(ctx)
 	fbo.log.CWarningf(ctx, "Unstaging branch %s after a resolution failure",
 		fbo.unmergedBID)
-	return fbo.unstageLocked(ctx, lState)
+	return fbo.unstageLocked(ctx, lState, true)
```

Comment:
Should use the named boolean (or enum).

Reply:
Actually I recently merged something that removes this whole function (`unstageAfterFailedResolution`) entirely, so it might be easier if you revert this change here and then rebase on master.